pkg/openstack/rabbitmq.go

```diff
@@ -38,6 +39,50 @@ const (
 	mqReady    mqStatus = iota
 )
 
+func findToDeleteRabbitMQs(
+	ctx context.Context,
+	instance *corev1beta1.OpenStackControlPlane,
+	helper *helper.Helper,
+) (rabbitmqv1.RabbitMqList, error) {
+
+	Log := GetLogger(ctx)
+	toDeleteList := &rabbitmqv1.RabbitMqList{}
+	// Fetch the list of RabbitMQ objects
+	rabbitList := &rabbitmqv1.RabbitMqList{}
+	listOpts := []client.ListOption{
+		client.InNamespace(instance.GetNamespace()),
+	}
+	err := helper.GetClient().List(ctx, rabbitList, listOpts...)
+	if err != nil {
+		Log.Error(err, "Could not get Rabbitmqs.")
+		return *toDeleteList, err
+	}
+
+	// Extract CR-Rabbit and Template-Rabbit names
+	crRabbitMQs := make(map[string]struct{})
+	templateRabbitsMap := make(map[string]struct{})
+
+	for _, rabbitObj := range rabbitList.Items {
+		crRabbitMQs[rabbitObj.Name] = struct{}{}
+	}
+
+	for name := range *instance.Spec.Rabbitmq.Templates {
+		templateRabbitsMap[name] = struct{}{}
+	}
+
+	Log.Info("", "CR Rabbits", fmt.Sprintf("%s", crRabbitMQs))
+	Log.Info("", "Template Rabbits", fmt.Sprintf("%s", templateRabbitsMap))
```

Comment:
Do we need these logs? They don't seem to provide any information about the status of the operation or what we want to delete. They simply log the same information that we already have in instance.Spec.Galera.Templates